### ISSUE TITLE
Could you add release v2.0.0 to the release page

### ISSUE THREAD

**Issue by geetotes** [Yelp/puppet-uchiwa#91]
I'm managing some puppet modules by hand and release number and I like to link back to the github release and I noticed that v2.0.0 isn't on there.

Thanks!

**Comment by bsmaldon** [comment]
I'm not sure if you might have already figured this out but I thought I'd share my workaround. I'm using librarian-puppet and I link to the specific commit I want. You'll be able to do a similar thing from the command line.

```
 mod "yelp/uchiwa",
    :git => 'https://github.com/Yelp/puppet-uchiwa.git',
    :ref => '92c7f71f864ecc2a1e747b1a48be69c737ec97a4'
```

**Comment by solarkennedy** [comment]
Added. Also released 2.1.0
